.github/workflows/continuous_integration.yml

```
@@ -38,29 +38,14 @@ jobs:
-          bundle install
-      - name: Run Tests
-        run: bundle exec rake
-      - name: standardrb
-        if:   matrix.standardrb == true
-        run:  bundle exec standardrb
-      - name: yard
-        if: matrix.yard == true
-        run:  bundle exec yard --fail-on-warning
+          ruby: ${{ matrix.ruby }}
+          appraisal: ${{ matrix.appraisal }}
+          bundler: ${{ matrix.bundler }}
```

actionlint (1.7.12)

[error] property "bundler" is not defined in object type {appraisal: string; ruby: string; standardrb: bool; yard: bool}

(expression)